e2e/src/usecases/Send.js

```diff
@@ -154,4 +159,226 @@ export default Send = () => {
+
+    it.todo('Then should display transaction as confirmed')
+
+    it.todo('Then should display correct transaction details')
+  })
+
+  describe('When multi-token send flow to phone number with one address (new flow)', () => {
+    beforeAll(async () => {
+      await device.uninstallApp()
+      await device.installApp()
+      await launchApp({
+        newInstance: true,
+        launchArgs: { statsigGateOverrides: `use_new_send_flow=true,use_viem_for_send=true` },
+      })
+      await quickOnboarding(SAMPLE_BACKUP_KEY_VERIFIED)
+    })
+
+    it('Then should navigate to send search input from home action', async () => {
+      await waitForElementByIdAndTap('HomeAction-Send', 30 * 1000)
+      await waitForElementId('SendSelectRecipientSearchInput', 10 * 1000)
+    })
+
+    it('Then should be able to enter a phone number', async () => {
+      await waitForElementByIdAndTap('SendSelectRecipientSearchInput', 30 * 1000)
+      await element(by.id('SendSelectRecipientSearchInput')).typeText(
+        SINGLE_ADDRESS_VERIFIED_PHONE_NUMBER
+      )
+      await element(by.id('SendSelectRecipientSearchInput')).tapReturnKey()
+      await isElementVisible('RecipientItem', 0)
+    })
+
+    it('Then tapping a recipient should show send button', async () => {
+      await element(by.id('RecipientItem')).atIndex(0).tap()
+      await waitForElementId('SendOrInviteButton', 30 * 1000)
+    })
+
+    it('Then tapping send button should navigate to Send Enter Amount screen', async () => {
+      await element(by.id('SendOrInviteButton')).tap()
+      await waitForElementId('SendEnterAmount/Input', 30 * 1000)
+    })
+
+    it('Then should be able to select token', async () => {
+      await element(by.id('SendEnterAmount/TokenSelect')).tap()
+      await element(by.id('cUSDSymbol')).tap()
+      await expect(element(by.text('cUSD')).atIndex(0)).toBeVisible()
+    })
+
+    it('Then should be able to enter amount and navigate to review screen', async () => {
+      await waitForElementByIdAndTap('SendEnterAmount/Input', 30 * 1000)
+      await element(by.id('SendEnterAmount/Input')).replaceText('0.01')
+      await element(by.id('SendEnterAmount/Input')).tapReturnKey()
+      await waitForElementByIdAndTap('SendEnterAmount/ReviewButton', 30 * 1000)
+      await isElementVisible('ConfirmButton')
+    })
+
+    it('Then should display correct recipient', async () => {
+      await expect(element(by.text(SINGLE_ADDRESS_VERIFIED_PHONE_NUMBER_DISPLAY))).toBeVisible()
+    })
+
+    it('Then should be able to send', async () => {
+      await element(by.id('ConfirmButton')).tap()
+      await enterPinUiIfNecessary()
+      await expect(element(by.id('errorBanner'))).not.toBeVisible()
+      await waitForElementId('HomeAction-Send', 30 * 1000)
+    })
+
+    it.todo('Then should display transaction as pending')
+
+    it.todo('Then should display transaction as confirmed')
+
+    it.todo('Then should display correct transaction details')
```

Comment:
todos

Reply:
I agree, if there's no active issue for adding these e.g. todo(ACT-xyz) we should just remove them.